Makefile

```diff
@@ -56,21 +55,22 @@ deps:
 POLKADOT_SUBMODULE    := $(DEPS_DIR)/substrate
 POLKADOT_RUNTIME_WASM := $(POLKADOT_SUBMODULE)/target/release/wbuild/node-template-runtime/node_template_runtime.compact.wasm
 
-deps-polkadot:
-	rustup update nightly
-	rustup target add wasm32-unknown-unknown --toolchain nightly
-	cargo install --git https://github.com/alexcrichton/wasm-gc
-
 # Useful Builds
 # -------------
 
-KOMPILE_OPTIONS :=
+KOMPILE_OPTIONS := --emit-json
 
 MAIN_MODULE        := KWASM-POLKADOT-HOST
 MAIN_SYNTAX_MODULE := KWASM-POLKADOT-HOST-SYNTAX
 MAIN_DEFN_FILE     := kwasm-polkadot-host
 
-SUBDEFN := kwasm
+ifneq (coverage,$(BUILD))
```

Comment:
Should this have been `ifeq`? The commands in the README no longer build the `kwasm` subdefiniton.

Reply:
@ehildenb ^^^

Reply:
I'll fix it in my next PR.